src/web/Button.tsx

```diff
@@ -157,7 +154,7 @@ export class Button extends React.Component<Types.ButtonProps, {}> {
         }
 
         if (this.props.disabled) {
-            buttonStyleMutations.opacity = 0.5;
+            buttonStyleMutations.opacity = this.props.disabledOpacity || 0.5;
```

Comment:
This also doesn't properly handle the case where disabledOpacity is specified as 0.

Reply:
Yes, thanks for catching it.